### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qwe123coder**
added 
https://forthebadge.com/images/badges/built-by-codebabes.svg

### CODE REVIEW

**Comment by CodingSpecies** [comment]
@qwe123coder , thank you so much for your contribution, though is it possible for you to perhaps change the badge, to  a more suitable one, like: [forthebadge](https://forthebadge.com)

`[forthebadge](https://forthebadge.com)`

thank you so much! :)

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Feel free to submit a pull request with your suggestions of awesome github profi
 [![made with markdown](https://forthebadge.com/images/badges/made-with-markdown.svg)](https://github.com/EddieHubCommunity)
 [![open source](https://forthebadge.com/images/badges/open-source.svg)](https://github.com/EddieHubCommunity)
 [![check it out](https://forthebadge.com/images/badges/check-it-out.svg)](https://github.com/EddieHubCommunity)
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
```
Comment (CodingSpecies) [inline README.md:13]: @qwe123coder Also, when you add it in, please change the `(https://forthebadge.com)` to `(https://github.com/EddieHubCommunity)` thank you!

**Comment by qwe123coder** [comment]
> @qwe123coder , thank you so much for your contribution, though is it possible for you to perhaps change the badge, to a more suitable one, for this repo like: [forthebadge](https://forthebadge.com)
> 
> `[forthebadge](https://forthebadge.com)`
 opps i put the worng on

**Review by CodingSpecies — APPROVED** [review]
LGTM! Thank you @qwe123coder, I will merge this! 👍🏼

**Pull request merged**